.github/workflows/update-deps.yaml

```
@@ -50,6 +61,17 @@ jobs:
             git diff && git status
             echo "HAS_GIT_DIFF=true" >> $GITHUB_OUTPUT
           fi
+      - name: Get Label
+        id: pr-label
+        if: ${{ steps.check_diff.outputs.HAS_GIT_DIFF == 'true' }}
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:3:39: Double quote to prevent globbing and word splitting

(shellcheck)